tensorboard/plugins/custom_scalar/tf_custom_scalar_dashboard/tf-custom-scalar-dashboard.html

```diff
@@ -0,0 +1,366 @@
+<link rel="import" href="../tf-utils/tf-utils.html">
+<link rel="import" href="tf-custom-scalar-card.html">
+
+<!--
+  A frontend that displays a custom set of line charts.
+-->
+<dom-module id="tf-custom-scalar-dashboard">
+  <template>
+    <tf-dashboard-layout>
+      <div class="sidebar">
+        <div class="sidebar-section">
+          <div class="line-item">
+            <paper-checkbox
+              checked="{{_showDownloadLinks}}"
+            >Show data download links</paper-checkbox>
+          </div>
+          <div class="line-item">
+            <paper-checkbox
+              checked="{{_ignoreYOutliers}}"
+            >Ignore outliers in chart scaling</paper-checkbox>
+          </div>
+          <div id="tooltip-sorting">
+            <div id="tooltip-sorting-label">Tooltip sorting method:</div>
+            <paper-dropdown-menu
+              no-label-float
+              selected-item-label="{{_tooltipSortingMethod}}"
+            >
+              <paper-menu class="dropdown-content" selected="0" slot="dropdown-content">
+                <paper-item>default</paper-item>
+                <paper-item>descending</paper-item>
+                <paper-item>ascending</paper-item>
+                <paper-item>nearest</paper-item>
+              </paper-menu>
+            </paper-dropdown-menu>
+          </div>
+        </div>
+        <div class="sidebar-section">
+          <tf-smoothing-input
+            weight="{{_smoothingWeight}}"
+            step="0.001"
+            min="0"
+            max="1"
+          ></tf-smoothing-input>
+        </div>
+        <div class="sidebar-section">
+          <tf-option-selector
+            id="x-type-selector"
+            name="Horizontal Axis"
+            selected-id="{{_xType}}"
+          >
+            <paper-button id="step">step</paper-button><!--
+            --><paper-button id="relative">relative</paper-button><!--
+            --><paper-button id="wall_time">wall</paper-button>
+          </tf-option-selector>
+        </div>
+        <div class="sidebar-section">
+          <tf-runs-selector selected-runs="{{_selectedRuns}}">
+          </tf-runs-selector>
+        </div>
+      </div>
+      <div class="center" id="categories-container">
+        <template is="dom-if" if="[[_dataNotFound]]">
+          <div class="no-data-warning">
+            <h3>The custom scalars dashboard is inactive.</h3>
+            <p>Probable causes:</p>
+            <ol>
+              <li>You haven't laid out the dashboard.
+              <li>You haven’t written any scalar data to your event files.
+            </ol>
+            <p>
+              To lay out the dashboard, pass a <code>Layout</code> protobuffer
+              to the <code>set_layout</code> method. For example,
+            </p>
+<pre>
+from tensorboard import summary
+from tensorboard.plugins.custom_scalar import layout_pb2
+...
+with tf.summary.FileWriter(LOGDIR) as file_writer:
+  file_writer.add_summary(
```

Comment:
I love this doc but it needs to be updated due to recent changes.

Reply:
Done. We now use `summary.custom_scalars_pb`.

Reply:
Yes but `FileWriter` isn't going to be used in `go/tf-summaries-2.0`. I guess we can change this doc later.

Reply:
OK, lets just leave out the part about constructing a `FileWriter`. Done.